src/lib/sql/sql_translator.cpp

```diff
@@ -1713,8 +1717,8 @@ std::shared_ptr<AbstractExpression> SQLTranslator::_translate_hsql_expr(
       Fail("Star expression should have been handled earlier");
 
     case hsql::kExprArrayIndex:
-    case hsql::kExprDatetimeField:
     case hsql::kExprHint:
+    case hsql::kExprCast:
       FailInput("Can't translate this hsql expression into a Hyrise expression");
```

Comment:
Not the most helpful error? Can't we output different failures for `HINT` and `CAST`?